### PR TITLE
feat: vyuh_cli

### PR DESCRIPTION
**Pull request by ProjectAJ14**
New Feature!!!

## vyuh_cli

A command-line interface for the Vyuh Framework to generate Vyuh projects from predefined templates.

project_create

```sh
Creates Vyuh projects, features, and sanity schemas.

Usage: vyuh create <subcommand> <project-name> [arguments]
-h, --help    Print this usage information.

Available subcommands:
  feature                 A Vyuh feature created by Vyuh CLI.
  feature-sanity-schema   A Vyuh feature Sanity schema created by Vyuh CLI.
  project                 A Vyuh Flutter project created by Vyuh CLI.

```

### CODE REVIEW

File: vyuh_cli/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## 0.0.1
+
+- First release with a set of core commands
+  - `vyuh create project`
+  - `vyuh create feature`
+  - `vyuh create feature-sanity-schema`
```
Comment (pavanpodila) [inline vyuh_cli/CHANGELOG.md]: we should just call it `schema` and then accept the CMS (as a parameter) later when we support multiple cms-es

File: vyuh_cli/lib/src/commands/create/utils/utils.dart [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+typedef MasonGeneratorFromBrick = Future<MasonGenerator> Function(Brick);
+
+final RegExp identifierRegExp = RegExp('[a-z_][a-z0-9_]*');
+final RegExp orgNameRegExp = RegExp(r'^[a-zA-Z][\w-]*(\.[a-zA-Z][\w-]*)+$');
+
+const String vyuhInfoText = '''
+
++----------------------------------------------------+
+| Looking for more features?                         |
+| We have an enterprise-grade solutions!             |
```
Comment (pavanpodila) [inline vyuh_cli/lib/src/commands/create/utils/utils.dart]: solution --- singular

Also, we can add a line (---) after the title line

File: vyuh_feature_sanity_schema/hooks/pubspec.lock [hunk 3]
```diff
@@ -5,10 +5,10 @@ packages:
     dependency: transitive
```
Comment (pavanpodila) [inline vyuh_feature_sanity_schema/hooks/pubspec.lock]: let's delete this file ... should not have been there

**Review by pavanpodila — CHANGES_REQUESTED** [review]

**Comment by pavanpodila** [comment]
Awesome piece of work @ProjectAJ14 🚀

File: vyuh_cli/CHANGELOG.md (hunk 1 above)
Comment (ProjectAJ14) [reply vyuh_cli/CHANGELOG.md]: Changed to "schema," the "cms" option has been added, with the default  CMS as "sanity."

File: vyuh_cli/lib/src/commands/create/utils/utils.dart (hunk 2 above)
Comment (ProjectAJ14) [reply vyuh_cli/lib/src/commands/create/utils/utils.dart]: Done

File: vyuh_feature_sanity_schema/hooks/pubspec.lock (hunk 3 above)
Comment (ProjectAJ14) [reply vyuh_feature_sanity_schema/hooks/pubspec.lock]: Done

**Review by pavanpodila — APPROVED** [review]

**Pull request merged**